### ISSUE TITLE
Suggestion: isomorphic API

### ISSUE THREAD

**Issue by lorensr** [helfer/minimongo-index#1]
Maybe `Collection.ensureIndex`

``` coffeescript
given collection, map

if Meteor.isServer
  collection._ensureIndex map

if Meteor.isClient
  fields = Object.keys map

  index = if fields.length is 1
    fields[0]
  else
    fields

  collection._collection._ensureIndex index
```

**Comment by teknologist** [comment]
+1

**Comment by helfer** [comment]
Good idea, but I think it might be better to just use `Collection._ensureIndex` for now so people don't have to rewrite their applications if they already have indices defined on the server.

**Comment by ElegantSudo** [comment]
@helfer just curious, does this still work? In theory I could just create indexes on both the client and server with Collection._ensureIndex.

**Comment by helfer** [comment]
Yes, it still works. I've been using minimongo-index with Meteor 1.3 and it still works. I'm sure it also works with 1.4, because there were no changes to minimongo.

In theory you could use _ensureIndex, but that's not how I implemented it. Feel free to make a PR though!
